notebooks/quickstart/data-owner/03-join-network.ipynb

```diff
@@ -0,0 +1,359 @@
+{
```

Comment:
<p>sounds incomplete: .. change it with any other available Network.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/6693/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>